usr/bin/rvc2mqtt.py

```diff
@@ -215,27 +223,33 @@ def getLine():
         if debug_level>0:
             print("{0:f} {1:X} ({2:X}) ".format(message.timestamp, message.arbitration_id, message.dlc),end='',flush=True)
 
-        canID = "{0:b}".format(message.arbitration_id)
-        prio  = int(canID[0:3],2)
-        dgn   = "{0:05X}".format(int(canID[4:21],2))
-        srcAD = "{0:02X}".format(int(canID[24:],2))
+        try:
+            canID = "{0:b}".format(message.arbitration_id)
+            prio  = int(canID[0:3],2)
+            dgn   = "{0:05X}".format(int(canID[4:21],2))
+            srcAD = "{0:02X}".format(int(canID[24:],2))
+        except:
+            dgn = None
+            if debug_level>0:
+                print(f"Failed to parse {message}")
 
-        if debug_level>0:
-            print("DGN: {0:s}, Prio: {1:d}, srcAD: {2:s}, Data: {3:s}".format(
-                dgn,prio,srcAD,", ".join("{0:02X}".format(x) for x in message.data)))
+        if dgn:
```

Comment:
Let's change the `if dgn:` to just `else:` of the `try:`... this way it'll only run if the `try` succeeds, and doesn't require the addition of another separate conditional check.

Reply:
Not sure what you mean by else of the the try. Do you mean move the whole block that's under if dgn: into the try block?